sdk/log/crate/src/logger.rs

```diff
@@ -217,6 +228,12 @@ macro_rules! impl_log_for_unsigned_integer {
                         };
 
                         // Number of available digits to write.
+                        //
+                        // JC: `offset` starts at max_digits, so if I write 1
+                        // digit, then offset will be `9`, which means available
+                        // would be `1`, which is not the number of available
+                        // digits, but rather the number of written digits. Am
+                        // I missing something?
```

Comment:
Yeah, the naming was not very clear. It is the number of "available" digits to write – a better name is `written` as you suggested.